npm/webpack-dev-server/src/loader.ts

```diff
@@ -59,22 +61,33 @@ export default function loader (this: unknown) {
   ctx.cacheable(false)
   const { files, projectRoot, supportFile } = ctx._cypress
 
-  const supportFileAbsolutePath = supportFile ? JSON.stringify(path.resolve(projectRoot, supportFile)) : undefined
-
-  return `
-  var loadSupportFile = ${supportFile ? `() => import(${supportFileAbsolutePath})` : `() => Promise.resolve()`}
+  const supportFileAbsolutePath = supportFile ? path.resolve(projectRoot, supportFile) : undefined
+  const supportFileRelativePath = supportFile ? path.relative(projectRoot, supportFileAbsolutePath || '') : undefined
+  const result = `
   var allTheSpecs = ${buildSpecs(projectRoot, files)};
 
   var { init } = require(${JSON.stringify(require.resolve('./aut-runner'))})
 
   var scriptLoaders = Object.values(allTheSpecs).reduce(
     (accSpecLoaders, specLoader) => {
       if (specLoader.shouldLoad()) {
-        accSpecLoaders.push(specLoader.load)
+        accSpecLoaders.push(specLoader)
       }
       return accSpecLoaders
-  }, [loadSupportFile])
+  }, [])
+
+  if (${!!supportFile}) {
+    var supportFile = {
+      absolute: "${supportFileAbsolutePath}",
+      relative: "${supportFileRelativePath}",
+      relativeUrl: "/__cypress/src/cypress-support-file.js",
```

Comment:
Providing the absolute & relative fields here was a change Zach proposed to align how files were queued up for loading in E2E and CT. Since much of this logic is really occurring outside the safety of TS it seemed like a good idea to align the contracts even though we're just using the `load` fn at the moment

Reply:
We do use the `relativeUrl` when making the fetch request (to get the source map), `absolute` and `relative` was just my attempt at making it symmetrical with e2e. When I was spiking into this I was running into an issue getting the paths and to work so I moved it from the reduce and hardcoded it. Definitely could be cleaned up a bit

Reply:
Right, I see. Definitely agree on aligning with E2E where possible.